ELearningPlatform/src/test/java/com/example/ELearningPlatform/repository/EnrollmentRepositoryTest.java

```diff
@@ -0,0 +1,51 @@
+package com.example.ELearningPlatform.repository;
+
+import com.example.ELearningPlatform.entity.Enrollment;
+import com.example.ELearningPlatform.model.EnrollmentStatus;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@DataJpaTest
+class EnrollmentRepositoryTest {
+    @Autowired
+    private EnrollmentRepository enrollmentRepository;
+    @Autowired
+    private TestEntityManager em;
+
+    @Test
+    void test_bulkUpdateEnrollmentsStatus() {
```

Comment:
wszedzie trzymasz ta konwencje nazewnicza? ze snake case? raczej bym pisal od should bo nie za czesto widze snake case w testach w iinnych projektach